app/models/user_review.rb

```diff
@@ -0,0 +1,7 @@
+class UserReview < ActiveRecord::Base
+  belongs_to :review
+  belongs_to :user
+
+  validates_presence_of :review, presence: true
+  validates_presence_of :user, presence: true
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.